tests/unit/files/rosters/dir/test1_us-east-2_test_badfile

```diff
@@ -0,0 +1,6 @@
+#!jinja|yaml
```

Comment:
I would prefer we create these tests when preparing the tests instead of storing them in the source code.
```python
@pytest.fixture
def create_roster_files(tmppath):
    contents = """
    #!jinja|yaml
    host: 127.0.0.2
    port: 22
    THIS FILE IS NOT WELL FORMED YAML
    sudo: true
    user: scoundrel
    """"
    with pytest.helpers.temp_file("test1_us-east-2_test_badfile", contents, directory=tmppath):
        .... etc ....
        yield